lib/srv6.c

```diff
@@ -485,5 +510,13 @@ json_object *srv6_locator_detailed_json(const struct srv6_locator *loc)
 		json_object_array_add(jo_chunks, jo_chunk);
 	}
 
+	/* set sids */
+	jo_sids = json_object_new_array();
+	json_object_object_add(jo_root, "sids", jo_sids);
+	for (ALL_LIST_ELEMENTS_RO(loc->sids, sidnode, sid)) {
+		jo_sid = srv6_locator_sid_detailed_json(loc, sid);
+		json_object_array_add(jo_sids, jo_sid);
+	}
+
```

Comment:
- separate lib change from test change (separate commit).
- this part of code should be made in the same commit as the changes done in show_srv6_locator_detail_cmd()